package.json

```diff
@@ -21,7 +21,7 @@
     "build": "npm run lint && npm run clean && npm run build:js",
     "build:js": "tsc --project tsconfig.build.json",
     "postbuild": "rimraf lib/tests",
-    "prepare:core": "git reset --hard && node utils/prepare-israeli-bank-scrapers-core.js && npm i --package-lock-only && npm ci && npm run lint -- --fix && npm run build",
+    "prepare:core": "git reset --hard && node utils/prepare-israeli-bank-scrapers-core.js && npm i --package-lock-only && npm ci && npm run lint:fix && npm run build",
```

Comment:
The code after "prepare:core" is not committed, I don't see a reason to lint:fix it.

Reply:
That make sense, removing